types/next-auth/jwt.d.ts

```diff
@@ -40,6 +40,10 @@ export interface JWTOptions {
     encryptionKey?: string;
     encode?(options: JWTEncodeParams): Promise<string>;
     decode?(options: JWTDecodeParams): Promise<WithAdditionalParams<JWT>>;
+    verificationOptions?: {
+      algorithms?: string[]
+      maxTokenAge?: string
```

Comment:
@balazsorban44 do you think `algorithms` and  `maxTokenAge` should both be optional? Or both are required?

Reply:
`verificationOptions` should be optional given [in the source code](https://github.com/nextauthjs/next-auth/blob/76df2b5e702be2084674c81e098d989c2ec39ed9/src/lib/jwt.js#L58-L61) we use a default parameter for it 📐

Regarding `algorithms` and `maxTokenAge`, [they're forwarded](https://github.com/nextauthjs/next-auth/blob/76df2b5e702be2084674c81e098d989c2ec39ed9/src/lib/jwt.js#L90) to [`jose`](https://www.npmjs.com/package/jose), looking at [its documentation ](https://github.com/panva/jose/blob/dd6b0eda9f07af21fb4d650fbc22ff5bd23bded4/docs/interfaces/jwt_verify.jwtverifyoptions.md) it seems both are optional 👍🏽

Reply:
Just a nitpick that according to [`jose` docs](https://github.com/panva/jose/blob/dd6b0eda9f07af21fb4d650fbc22ff5bd23bded4/docs/interfaces/jwt_verify.jwtverifyoptions.md), `maxTokenAge` is actually `string | number` 🏂

Reply:
@lluia thanks! Updated 🙇